src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/builder_priority_tag.xml

```diff
@@ -29,14 +29,14 @@
       <tag name="tag2"/>
     </tags>
   </definition>
-  <definition id="definition_4" class="Full\Qualified\Class4" public="true" synthetic="true" lazy="false" shared="true" abstract="false" autowired="false" autoconfigured="false" file="/path/to/file">
+  <definition id="definition_4" class="Full\Qualified\Class4" public="true" synthetic="true" lazy="false" shared="true" abstract="false" autowired="false" autoconfigured="false" deprecated="false" file="/path/to/file">
```

Comment:
The deprecated attribute is false by default. Why explicitly setting it?

Reply:
I had to patch all tests otherwise it failed @GromNaN

Reply:
all good, the dump is meant to reflect the full service definition.